src/Core/Domain/Shop/ValueObject/ShopId.php

```diff
@@ -40,10 +40,12 @@ class ShopId
 
     /**
      * @param int $shopId
+     *
+     * @throws ShopException
      */
-    public function __construct($shopId)
+    public function __construct(int $shopId)
```

Comment:
@jolelievre BC Break

Reply:
Mmm, I have doubts on this one, because this VO was introduced in 177 but it's not used anywhere
It was probably used in a previous implementation and not cleaned after a refacto..
The thing is it was shipped in the RC1 though..

Reply:
If it's 177, it's ok.